srcpkgs/python3-etebase/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'python3-etebase'
+pkgname=python3-etebase
+version=0.31.5
+revision=1
+archs="x86_64"
```

Comment:
why?

Reply:
because it throws error[E0107] on all other architectures.
I am trying to figure out how to fix it but i am no rust specialist so ¯\_(ツ)_/¯

Reply:
Oh damn! Thank you a lot! Had no clue it was a python issue. Will do!

Reply:
It's not a Python issue; the cargo failure is the same as affects some other templates. I just grepped for "indexmap" in `srcpkgs/*/template` and found several that updated `autocfg`. The other changes I made just bring the template in line with other packages that use `python3-setuptools-rust`.

I'm rolling this into the pending Py3.11 update just to keep conflicts down; if there's a pressing need we can merge the fix immediately, but otherwise I'll just wait.

Reply:
There's no immediate need, no, it can wait a few weeks no problem. Would you like me to create a new pull request with all the changes recommended in the thread , or wait till 3.11 upgrade?

Reply:
I'll just carry the patch for now; no action is required on your part.